dashboards-notebooks/public/components/notebook.tsx

```diff
@@ -633,10 +634,26 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
     })
   }
 
+  configureViewParameter(id: string) {
+    const url = new URL(window.location);
+    url.searchParams.set('view', id);
+    window.history.pushState({}, '', url);
+  }
+
   componentDidMount() {
     this.setBreadcrumbs('');
     this.loadNotebook();
     this.checkIfReportingPluginIsInstalled();
+    const url = new URL(window.location);
+    if (url.searchParams.get('view') === null) {
+      this.configureViewParameter('view_both');
+    }
+    if (window.location.href.includes('output_only')) {
+      this.setState({selectedViewId: 'output_only'});
+    }
+    else if (window.location.href.includes('input_only')) {
```

Comment:
change these to `url.searchParams.get('view') === 'output_only'` instead of string includes?

Reply:
good catch, will change